graph/client.py

```diff
@@ -130,12 +152,19 @@ def parse_arguments(self, args):
             action="store_true",
             default=False,
             help="It does not start the web browser.")
+        parser.add_argument(
+            '--tree',
+            action="store_true",
+            default=False,
+            help=("Switch graph_html_interpreter to tree_html_interpreter."
+                  " Rule will interpret as directory tree."))
         parser.add_argument(
             '--remove-pass-tests',
             action="store_true",
             default=False,
-            help=('Do not display passing tests for better orientation in'
-                  ' graphs that contain a large amount of nodes.(Not implemented)'))
+            help=(
+                'Do not display passing tests for better orientation in'
+                ' graphs that contain a large amount of nodes.(Not implemented)'))
```

Comment:
I have noticed that you mix single quotes and double quotes in your code. Even if they have roughly the same meaning, please try to be more consistent in using the quotes.